ang/civicase/case/details/directives/case-details.directive.js

```diff
@@ -234,10 +241,30 @@
       });
     };
 
+    /**
+     * Listener for civicase::activity-details::affix-initialised
+     */
+    function activityDetailsInitListener () {
```

Comment:
I think it's better to name this function `activityDetailsAffixListener` or similar.